src/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core/Models/DevCanvasGeneratorInfo.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core.Models
+{
+    /// <summary>
+    /// Describes the generator that produced an insight.
+    /// </summary>
+    public class DevCanvasGeneratorInfo : ToolComponent
```

Comment:
Is this class a duplicate of our nuget?  How much of our content that is for server would be useful to reuse here?

Reply:
I believe it is a duplicate of what's in our repo. Not sure if there's an easy way to reuse things since exposing the nuget publicly is probably not worth the effort